src/UI/Internal/Text.elm

```diff
@@ -276,3 +262,29 @@ mobileAttributes size =
             , Font.letterSpacing 2
             , Font.extraBold
             ]
+
+
+mapOpt : (Options -> Options) -> Text -> Text
+mapOpt applier (Text spans) =
+    spans
+        |> List.map (spanMapOpt applier)
+        |> Text
+
+
+spanMapOpt : (Options -> Options) -> Span -> Span
```

Comment:
same above.

One thing I have to mention, I'm not a huge fan of abbreviations, for example, using `opt` instead of `option`, because in general, I don't see any benefit, but I now that you like a lot, so we have to balance :D (just a thought)